doc/source/index.rst.template

```diff
@@ -52,3 +53,66 @@ See the :ref:`overview` for more detail about what's in the library.
     development/index
     whatsnew/index
     {% endif -%}
+
+
+* :doc:`whatsnew/v0.24.1`
```

Comment:
is this right?

Reply:
It's the link we currently have on the main page (I only omitted the subsections for this page)

Reply:
Ah, I suppose you meant the "0.24.1". That should indeed be 0.25.0 for master (and needs to be changed to 0.24.2 on the 0.24.x branch), thanks for the catch.